src/index.html

```diff
@@ -15,8 +15,92 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
```

Comment:
The 'href' attribute in the logo link is set to '#logo', which does not correspond to any section within the current HTML document. Consider updating it to a valid section ID or URL to ensure proper navigation.